react/introduction/fundamentals/react-what-is-state.md

```diff
@@ -120,4 +120,4 @@ Where should **initial state** be set in a component class?
 This has to do with how data flows in React. The general idea behind this concept is that data goes down while actions go up, effectively creating a circular system. This will be explained in more detail and the end of the workout.
```

Comment:
```suggestion
This has to do with how data flows in React. The general idea behind this concept is that data goes down while actions go up, effectively creating a circular system. This will be explained in more detail at the end of the workout.
```